pkg/sass-parser/lib/src/stringifier.ts

```diff
@@ -67,7 +70,67 @@ export class Stringifier extends PostCssStringifier {
     )(statement, semicolon);
   }
 
+  private ['debug-rule'](node: DebugRule, semicolon: boolean): void {
+    this.builder(
+      '@debug' +
+        (node.raws.afterName ?? ' ') +
```

Comment:
I just realized that setting `afterName = ''` in the case of `@debug foo` would result in `@debugfoo`. I was worried about that, but then I tried it on postcss and that's the same behavior! o_O

Interestingly, in postcss, if I set `aftername = null`, this results in `@debugnullfoo` 🤔